docfx/en/components/toc.json

```diff
@@ -621,7 +621,7 @@
     ]
   },
   {
-    "exclude": ["Angular", "React"],
+    "exclude": ["Angular"],
     "name": "Tree Grid",
     "href": "grids/tree-grid/overview.md",
     "status": "",
```

Comment:
Update the status of the Tree Grid Overview to NEW_REACT. 

`"status": ["NEW_REACT"]`